var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -263,6 +269,12 @@ def edit(self, spec, prefix):
                 '-ffree-line-length-none',
                 '-ggdb',  # make sure we get proper Fortran backtraces
             ]
+        elif '%aocc' in spec:
+            fcflags += [
+                '-D__F2008',
```

Comment:
is this preprocessor flag used by the compiler?

Reply:
Not necessarily required for the compiler, removing  `-D__F2008`.